client/rootReducer.js

```diff
@@ -0,0 +1,15 @@
+import { combineReducers } from 'redux';
+
+import recipes from './reducers/recipeReducer';
```

Comment:
default imports should match the base filename e.g.  import `recipeReducer` from './reducers/`recipeReducer`'